src/Jellyfin.MediaEncoding.Keyframes/FfProbe/FfProbeKeyframeExtractor.cs

```diff
@@ -62,12 +62,17 @@ internal static KeyframeData ParseStream(StreamReader reader)
             var rest = line[(firstComma + 1)..];
             if (lineType.Equals("packet", StringComparison.OrdinalIgnoreCase))
             {
-                if (rest.EndsWith(",K_"))
+                // Split time and flags from the packet line. Example line: packet,7169.079000,K_
+                var secondComma = rest.IndexOf(',');
+                var pts_time = rest[..secondComma];
```

Comment:
```suggestion
                var pts_time = rest[..secondComma];
```

Please use a more descriptive variable name and remove snake_case.

Reply:
I'll change it to camelCase. What would you prefer for the variable name? Because ptsTime is the name of the field from ffprobe as listed in the arguments above:
``-show_entries packet=pts_time,flags``